README.md

```diff
@@ -158,10 +158,130 @@ This will determine any missing `ansible-core` tags and create them in `ansible-
 nox -s tag
 
 # If you use a different upstream remote, specify the name.
-nox -s tag -- --remote <name> tag 
+nox -s tag -- --remote <name> tag
 
 # If your core repo is not in the same filesystem location, specify the path.
 nox -s tag -- --core <path> tag
 ```
 
 See `nox -s tag -- --help` for extended options.
+
+## Branching for new stable versions
+
+The branching strategy for this repository mirrors the [`ansible/ansible`](https://github.com/ansible/ansible) repository.
+When a new `stable-*` branch is created in the core repository, a corresponding branch in the `ansible-documentation` repository needs to be created.
+There are various other changes that should occur around the same time that the new stable branch is cut.
+
+### Creating stable branches
+
+Someone with maintainer access must create the new branch.
+You can do so as follows:
+
+```bash
+# Make sure your checkout is up to date.
+git fetch upstream
+
+# Create a new stable branch against the devel branch.
+git checkout -b stable-2.18 upstream/devel
+
+# Push the new stable branch to the repository.
+git push upstream stable-2.18
+```
+
+After the new stable branch is created, the following changes should be committed as pull requests to the new stable branch:
+
+* Update the core branch in the `docs/ansible-core-branch.txt` file.
+* Remove devel-only tooling.
+* Update Python versions in the support matrix.
+
+### Updating the core branch
```

Comment:
Nit: Some headings use gerunds (e.g., "Creating stable branches") and other headings and the checklist are imperative (e.g., "Remove devel-only tooling"). It'd probably be better to be consistent.

Reply:
Yeah, good catch. Thanks.